### ISSUE TITLE
input type=button with aria-pressed does not work

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/standards-support#223]
### Summary

input type=button with aria-pressed does not work

1. save as HTML file:
```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>aria-pressed</title>
	</head>
<body>

	<a href="#">test</a>
	<input type='button' value='Stop' aria-pressed='true'>
	<input type='button' value='Start' aria-pressed='false'>
	<button aria-pressed=true>On</button>
	<button aria-pressed=false>Off</button>

</body>
</html>
```
2. Start JAWS and IE 11
3. tab to the button, activate the buttons or read with arrow keys, navigate with B and look at the entries in the element list (CRTL+INS+B)

### Expected result

All buttons are output as toggle buttons

### Actual result

Correct output for button element. No output with input type=button. With Tab the focus remains on the previous element and when activating the button the previous element is clicked

Problem only occurs in IE 11 (not with Firefox, Chrome, Edge)

#### JAWS version and build number

JAWS 2019.1906.10
 
#### Operating System and version

Windows 10, Windows 8

#### Browser and version:

Internet Explorer 11.0.9600.19400

**Comment by lamador** [comment]
The issue was reproducible using the latest version of JAWS (2020.2006.12) and IE11 (11.836.18362.0) on Windows 10. The "Stop" and "Start" button names are not announced and navigating using the Virtual PC Cursor ignores these buttons.

**Comment by stevefaulkner** [comment]
closing as wontfix as we are no longer supporting IE11
